source/agora/consensus/data/Transaction.d

```diff
@@ -40,6 +46,9 @@ import std.algorithm;
 
 public struct Transaction
 {
+    /// Transation Type(Payment, Freeze)
```

Comment:
```suggestion
    /// Transation type
```

I suggest removing the stuff in the parentheses because they will go out of date quickly when we add new types to `TxType`. This comment will be forgotten about.

Reply:
Suggested change